ui/pages/prelogin/index.js

```diff
@@ -0,0 +1,35 @@
+import LoginLayout from '../../components/layouts/login'
+import Cookies from 'universal-cookie'
+
+export default function PreLogin() {
+  const cookies = new Cookies()
+  const organizations = cookies.get('orgs')
+
+  return (
+    <>
+      <h1 className='text-base font-bold leading-snug'>Welcome to Infra</h1>
+      <h2 className='my-1.5 mb-4 max-w-md text-center text-xs text-gray-400'>
+        Choose your organization to login to.
+      </h2>
+      {organizations?.length > 0 && (
```

Comment:
Don't think we need this check here since the `organizations?.map` will just render nothing